hw/ip/otp_ctrl/data/otp_ctrl.prj.hjson

```diff
@@ -8,10 +8,23 @@
     dv_doc:             "../doc/dv",
     hw_checklist:       "../doc/checklist",
     sw_checklist:       "/sw/device/lib/dif/dif_otp_ctrl",
-    version:            "0.1",
-    life_stage:         "L1",
-    design_stage:       "D2",
-    verification_stage: "V2",
-    dif_stage:          "S1",
-    notes:              "",
+    revisions: [
+      {
+        version:            "0.1",
+        life_stage:         "L1",
+        design_stage:       "D2",
+        verification_stage: "V2",
+        dif_stage:          "S1",
+        commit_id:          "127b109e2fab9336e830158abe449a3922544ded",
+        notes:              "",
+      }
+      {
+        version:            "0.2",
+        life_stage:         "L1",
+        design_stage:       "D2",
```

Comment:
Hey @msfschaffner, will design be stable enough to stay D2 or shall I roll back to D1 as well?

Reply:
Yes, design wise this should remain stable enough to stay in D2.

Reply:
Are we ok with version bump to 0.2? Shouldn't we be closer to 1.0?

Reply:
Good point! Given that we're working towards the final feature set for the tapeout, this could technically be 1.0 now according to https://docs.opentitan.org/doc/project/development_stages/#versioning.
We should probably align all other IPs to 1.0 as part of their next development stage transitions as well (CC @tjaychen @eunchan).

Reply:
thanks for bringing this up! I will change this PR to 1.0